src/dotnet/Fable.JS/bench/splitter.config.js

```diff
@@ -1,12 +1,14 @@
-const babelOptions = {
+const fableUtils = require("fable-utils");
+
+const babelOptions = fableUtils.resolveBabelOptions({
```

Comment:
It's not necessary any more to resolve the paths because Fable copies fable-core and sources from Nuget packages into the current directory (in the hidden `.fable` folder) to avoid problems :)

Reply:
but I'll take it out from here, as it works either way.

Reply:
a minute too late, already merged :) but you can take it out if not needed.